provisioning/device/samples/register_x509.js

```diff
@@ -35,6 +39,22 @@ deviceClient.register(function(err, result) {
     console.log('registration succeeded');
     console.log('assigned hub=' + result.assignedHub);
     console.log('deviceId=' + result.deviceId);
+    var connectionString = 'HostName=' + result.assignedHub + ';DeviceId=' + result.deviceId + ';x509=true';
+    var hubClient = Client.fromConnectionString(connectionString, iotHubTransport);
+    hubClient.setOptions(deviceCert);
+    hubClient.open(function(err) {
+      if (err) {
+        console.error('Failure opening iothub connection: ' + err.message);
+      } else {
+        console.log('Client connected');
+        var message = new Message('Hello world');
+        hubClient.sendEvent(message, function(err, res) {
+          if (err) console.log('send error: ' + err.toString());
+          if (res) console.log('send status: ' + res.constructor.name);
+          process.exit(1);
+        });
+      }
+    });
```

Comment:
>}); [](start = 4, length = 3)

These changes are already in master, is it really needed to move this right now.  Eventually this will be fi'd into here.

Reply:
bring those commits by accidents. I'm cancelling this PR and start a new one with just the required samples.